trie/secure_trie.go

```diff
@@ -161,18 +160,20 @@ func (t *StateTrie) UpdateAccount(address common.Address, acc *types.StateAccoun
 	if err != nil {
 		return err
 	}
-	if err := t.trie.TryUpdate(hk, data); err != nil {
+	if err := t.trie.Update(hk, data); err != nil {
 		return err
 	}
 	t.getSecKeyCache()[string(hk)] = address.Bytes()
 	return nil
 }
 
-// Delete removes any existing value for key from the trie.
-func (t *StateTrie) Delete(key []byte) {
-	if err := t.DeleteStorage(common.Address{}, key); err != nil {
-		log.Error("Unhandled trie error in StateTrie.Delete", "err", err)
-	}
+// MustDelete removes any existing value for key from the trie.
+//
+// This function will panic if any error is encountered.
```

Comment:
Will it really? Won't it just log the error?

Reply:
I forgot to change the comment, will fix it.